api/v1/errors/SqlError.js

```diff
@@ -0,0 +1,28 @@
+function checkError(err) {
```

Comment:
Yeah, because if there isn't a referenced row, it throws some kind of error, duplicate entries are handled by bookshelf and then everything else is handled by checkit and our defined error types